server/handlers/postSurveys.js

```diff
@@ -1,91 +1,91 @@
-// const Joi = require('@hapi/joi');
+const Joi = require('@hapi/joi');
+
+const passport = require('passport');
 const addQuestions = require('../queries/addQuestions');
-const createSurvey = require('../queries/createSurvey');
+const addSurvey = require('../queries/addSurvey');
+
+const postSurveys = async (req, res, next) => {
+  passport.authenticate('jwt', { session: false }, async (err, user, info) => {
+    if (err) {
+      throw new Error(err);
+    }
+    if (info) {
+      return res.status(401).json({ message: info.message });
+    }
+    if (user) {
+      // eslint-disable-next-line no-unused-vars
+      const surveyObjectSchema = Joi.object().keys({
```

Comment:
This JOI validation is quite big. Maybe it could be moved to a different file?

Reply:
yes, will put on a different file under a folder called schema